docs/sites/breaking.rst

```diff
@@ -0,0 +1,10 @@
+Breaking changes
```

Comment:
I'd like to talk about major releases here as well. We should either (1) commit to never _actually_ making breaking changes without releasing a new major version or, at least, (2) promise a warning for X minor releases/Y months before we make the breaking change. I'd personally propose to go for (1).

Reply:
(1) sounds great. For nullability, we will do (2) though, right? So I'm not sure I can put (1) in the text xD

Reply:
I think it would be fine to do a major release in a couple months 🤔 but happy to talk about this, we definitely shouldn't document something that we're going to deviate from though :smile: